src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -63,6 +64,10 @@ public function process(ContainerBuilder $container)
             $container->getDefinition('twig.extension.code')->replaceArgument(0, $container->getParameter('templating.helper.code.file_link_format'));
         }
 
+        if (!$container->has('security.token_storage')) {
```

Comment:
I'm sure :) If the token storage does not exist, it means you are using an older version, where we need to inject the security service.